src/directLine.ts

```diff
@@ -53,14 +53,26 @@ export interface UnknownMedia{
 
 export type CardActionTypes = "call" | "downloadFile" | "imBack" | "messageBack" | "openUrl" | "playAudio" | "playVideo" | "postBack" |  "signin" | "showImage";
 
-export interface CardAction {
+export type CardAction =  CardActionWithImage | CardActionWithTitle;
```

Comment:
Double space after equal sign.

```suggestion
export type CardAction = CardActionWithImage | CardActionWithTitle;
```